plugins/importexport/crossref/filter/ArticleCrossrefXmlFilter.inc.php

```diff
@@ -162,10 +179,26 @@ function createJournalArticleNode($doc, $submission) {
 		}
 
 		// abstract
-		if ($abstract = $publication->getData('abstract', $locale)) {
-			$abstractNode = $doc->createElementNS($deployment->getJATSNamespace(), 'jats:abstract');
-			$abstractNode->appendChild($node = $doc->createElementNS($deployment->getJATSNamespace(), 'jats:p', htmlspecialchars(html_entity_decode(strip_tags($abstract), ENT_COMPAT, 'UTF-8'), ENT_COMPAT, 'UTF-8')));
-			$journalArticleNode->appendChild($abstractNode);
+		$abstractLanguages = array_keys($publication->getData('abstract'));
```

Comment:
@ctgraham, this part is slightly different from the changes in main and 3_4_0 -- It considers article locale at the first place and the maximum of 10 entries. Shell the other branches consider this too?

Reply:
I believe 3.3.0 still uses Crossref schema 4.3.6, which limits the number of abstracts to 10.  Crossref schema 5.3.1 has no limit on the number of abstracts, so this logic is not needed on 3.4 and later.